Makefile

```diff
@@ -1,7 +1,7 @@
 CXX          ?= g++
 CXXOPTS      ?= -g -Wall -Werror -std=c++11 -Iinclude/ -Ideps/url-cpp/include
```

Comment:
Typically, `-g` would be specified in just the `DEBUG_OPTS` and not in the `RELEASE_OPTS`. It appears you're passing it to both.